src/perf_capabilities.ml

```diff
@@ -41,10 +41,13 @@ module Version = struct
 end
 
 let supports_configurable_psb_period () =
-  let cyc_cap =
-    In_channel.read_all "/sys/bus/event_source/devices/intel_pt/caps/psb_cyc"
-  in
-  String.( = ) cyc_cap "1\n"
+  try
+    let cyc_cap =
+      In_channel.read_all "/sys/bus/event_source/devices/intel_pt/caps/psb_cyc"
+    in
+    String.( = ) cyc_cap "1\n"
+  with
+  | _ -> false
```

Comment:
Worth leaving a comment as to why. I'm assuming it's because this file doesn't exist on AMD? (In which case, I think we should be matching on the `ENOENT` error only.)